scripts/mkcloud

```diff
@@ -1220,8 +1220,9 @@ Optional
     TESTHEAD='' | 1  (default='')
         use Media from Devel:Cloud:Staging and add test update repositories (except for GM* targets)
     cephvolumenumber  (default=0)
-        the number of 11GB ceph volumes that will be created per node
-        note: proposal step does NOT contain a ceph proposal, do it manually
+        the number of 21GB ceph volumes that will be created per node
```

Comment:
Not quite right - it's 21GB by default but can be changed by setting `cephvolume_hdd_size`.